<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="temp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>温度曲线</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serial">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>串口通信</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fluorescence">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>荧光图表</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1024</width>
      <height>500</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>500</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="serial_widget">
     <widget class="QComboBox" name="bound_select">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>0</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="port_select">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>端口选择</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>0</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>波特率</string>
      </property>
     </widget>
     <widget class="QPushButton" name="serial_link">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
     <widget class="QPushButton" name="serial_disconnect">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>0</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>断开</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_receive">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>80</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>清除接收</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="receive_select">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>40</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>接收开关</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QScrollArea" name="receive_Area">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>820</width>
        <height>320</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>820</width>
        <height>320</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>818</width>
         <height>318</height>
        </rect>
       </property>
       <widget class="QPlainTextEdit" name="Receive_msg">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>801</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLineEdit" name="send_msg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>700</width>
        <height>50</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="send_button">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>390</y>
        <width>90</width>
        <height>50</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="temp_widget">
     <widget class="QChartView" name="chart_view">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>791</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="temp_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>temp_1</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="temp_set">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>160</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>110</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>温度设定：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="temp_set_button">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>160</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="temp_start_button">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>50</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>1120</width>
        <height>300</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>扩增使能开关</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="floure_widget"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
